plugins/generator-1.19.2/forge-1.19.2/templates/elementinits/items.java.ftl

```diff
@@ -99,6 +103,36 @@ public class ${JavaModName}Items {
 		return REGISTRY.register(block.getId().getPath(), () -> new DoubleHighBlockItem(block.get(), new Item.Properties().tab(tab)));
 	}
 	</#if>
+
+	<#if w.hasItemsWithCustomProperties()>
+	<#compress>
+	@SubscribeEvent public static void clientLoad(FMLClientSetupEvent event) {
+		event.enqueueWork(() -> {

```

Comment:
Maybe the fact of calling this method does not let these providers run normally in MP worlds?

Reply:
FMLClientSetupEvent - this will only run on client as properties are on client side.

This is normal separation of client and server side logic MInecraft does for a while now

Reply:
Only data that is synced properly from server to client side will work with property conditions, similar to what things will be shown properly in eg. dynamic GUI labels